Maquetado/scss/grid.scss

```diff
@@ -1,12 +1,14 @@
-.tech-container-grid {
-  display: grid;
-  grid-template-columns: 1fr 1fr 1fr;
-  grid-template-rows: auto;
-  grid-template-areas:
-    "tech-a tech-b tech-c"
-    "tech-a tech-d tech-e";
-  margin-bottom: 70px;
-  padding: 0 300px;
+@media screen and (min-width: 1024px) {
+  .tech-container-grid {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: auto;
+    grid-template-areas:
+      "tech-a tech-b tech-c"
+      "tech-a tech-d tech-e";
+    margin-bottom: 70px;
+    padding: 0 300px;
```

Comment:
this breaks the layout for sure, the screen has a ~320px width  and this way we have 600px in padding.